src/test/java/com/streisky/virtualshowcasebackend/domain/account/repository/AccountRepositoryTest.java

```diff
@@ -0,0 +1,33 @@
+package com.streisky.virtualshowcasebackend.domain.account.repository;
+
+import java.util.Optional;
+
+import com.streisky.virtualshowcasebackend.domain.account.entity.Account;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

Comment:
Same question for @ActiveProfiles("test") annotations.

Reply:
The "AutoConfigureTestDatabase" is to use a real database and not be replaced by memory database. In beginning of tests implementation I was using a real database, but after I switch and forgot to remove this anotation.

The "ActiveProfiles" is necessary to define a active profile in tests. Because this annotation, the spring acess application-test.properties intead of application.properties.

Reply:
I not create a application.properties in the tests folder. I created in the resources folder of project. Because this is necessary define the active profile I guess. 
There are two different approaches I imagine, but I will keep this way as it is working correctly.